policies.go

```diff
@@ -948,19 +948,37 @@ func (host selectedHostPoolHost) Mark(err error) {
 }
 
 type dcAwareRR struct {
-	local           string
-	localHosts      cowHostList
-	remoteHosts     cowHostList
-	lastUsedHostIdx uint64
+	local            string
+	localHosts       cowHostList
+	remoteHosts      cowHostList
+	lastUsedHostIdx  uint64
+	permitDCFailover bool
+}
+
+type dcFailoverDisabledPolicy interface {
```

Comment:
Let's change it's naming because it is common for all dc aware policies, and their options 
```suggestion
type dcAwareHostPolicy interface {
```

Reply:
but it is also for rack aware policies, naming it dcAware* might be confusing